presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -44,6 +44,11 @@ class TaskManager {
     nodeId_ = nodeId;
   }
 
+  void setBaseSpillDirectory(const std::string& baseSpillDirectory) {
+    VELOX_CHECK(!baseSpillDirectory.empty());
```

Comment:
Shall we also check baseSpillDirectory_ is empty?

Reply:
We are checking if baseSpillDirectory_ is empty already. Did you mean something else?